labelparse.py

```diff
@@ -13,21 +13,17 @@
                   'terraform-newrelic-apm', 'newrelic-cloudformation', 'newrelic-cli', 'newrelic-kubernetes-operator', 'newrelic-client-go', 'developer-toolkit']
 
 
-def print_labels(issue):
-    '''Loops through repo data'''
-
-    if 'labels' in issue and issue['labels'] != []:
-        for label in issue['labels']:
-            print('Label: ' + label['name'])
-
-
 def print_issue(issue):
     '''prints issues'''
-    print('\n')
-    print('Repo: ' + issue['repository_url'])
-    print('ID: ' + str(issue['id']))
-    print('Title: ' + issue['title'])
-    print_labels(issue)
+
+    labels = ', '.join([label['name'] for label in issue.get('labels', [])])
+
+    print(f"""
+Repo: {issue['repository_url']}
+ID: {issue['id']}
+Title: {issue['title']}
+Labels: {labels}
+    """)
```

Comment:
This is a little more pythonic than multiple print lines for a couple reasons:

1. It uses Python3's f-string for interpolation (removing the need to cast `issue['id']` to a string
2. It only calls `print()` once